Core/AssemblyMetadata/AssemblyMetadataParser.cs

```diff
@@ -241,7 +241,7 @@ public PrimitiveTypeCode GetUnderlyingEnumType(string type)
                     }
                 }
 
-                throw new UnknownTypeException();
+                throw new UnknownTypeException(@"Type {type} is of uknown TypeCode");
```

Comment:
Small typo here ;)

Reply:
Maybe pass the Type as parameter to the exception?

Reply:
typo**s** ;)

```suggestion
                throw new UnknownTypeException($"Type {type} is of unknown TypeCode");
```